lib/decode.c

```diff
@@ -916,3 +916,19 @@ quirc_decode_error_t quirc_decode(const struct quirc_code *code,
 
 	return QUIRC_SUCCESS;
 }
+
+void quirc_flip(struct quirc_code *code)
+{
+    struct quirc_code flipped;
+    memset(&flipped, 0, sizeof(flipped));
```

Comment:
nit: We mandate c99 support, so you could write `struct quirc_code flipped = {0};` instead. In practice, I believe most compilers will generate the same output for both cases, so it's only a matter of readability.